inc/dashboardcards.class.php

```diff
@@ -0,0 +1,642 @@
+<?php
+
+/**
+ * -------------------------------------------------------------------------
+ * Credit plugin for GLPI
+ * -------------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of Credit.
+ *
+ * Credit is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Credit is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Credit. If not, see <http://www.gnu.org/licenses/>.
+ * -------------------------------------------------------------------------
+ * @author    François Legastelois
+ * @copyright Copyright (C) 2017-2023 by Credit plugin team.
+ * @license   GPLv3 https://www.gnu.org/licenses/gpl-3.0.html
+ * @link      https://github.com/pluginsGLPI/credit
+ * -------------------------------------------------------------------------
+ */
+
+use CommonDBTM;
+use PluginCreditEntity;
+use PluginCreditTicket;
+use QueryExpression;
+use Ticket;
+use Session;
+use Glpi\Dashboard\Provider;
+
+class PluginCreditDashboardCards extends CommonDBTM
```

Comment:
Here is a list of credits (2 on the root entity and one on the Subentity entity)

![image](https://github.com/user-attachments/assets/f95d1bb6-21b5-4799-a00a-eb00bb2e1f92)


In the root tree view, 

![image](https://github.com/user-attachments/assets/fb9d936e-17c4-4596-adbd-ed4c18eaa4d9)

I should see 32 Initial quantity of credit, but from dashbaord sub entity are not take into account

![image](https://github.com/user-attachments/assets/0777f512-a1e7-49f7-ad1a-45c09fea9f19)